Client/Frontend/Home/ActivityStreamTopSitesCell.swift

```diff
@@ -143,8 +143,9 @@ class TopSiteItemCell: UICollectionViewCell {
             contentView.backgroundColor = suggestedSite.backgroundColor.isBlackOrWhite ? UIColor.whiteColor() : suggestedSite.backgroundColor
         } else {
             guard let url = site.icon?.url, favURL = NSURL(string:url) else {
-                contentView.backgroundColor = UIColor.lightGrayColor()
-                imageView.image = FaviconFetcher.getDefaultFavicon(site.tileURL)
+                let siteURL = NSURL(string: site.url)!
```

Comment:
We have this neat extension on `String` called `asURL!` instead of having to do this if you prefer.